views/components/ConfigurationCategoryCard/index.jsx

```diff
@@ -6,43 +6,29 @@ import {
   CardContent,
   Divider,
   IconButton,
-} from '@material-ui/core'
-import Delete from '@material-ui/icons/Delete'
-import Copy from '@material-ui/icons/FileCopy'
+} from '@mui/material'
+import Delete from '@mui/icons-material/Delete'
+import Copy from '@mui/icons-material/FileCopy'
 
 const ConfigurationCategoryCard = ({
   children,
-  classes,
+
   formIndex,
   onCopy,
   onRemove,
   rowNum,
   width,
 }) => {
   return (
-    <Card
-      className={classes.configurationCategoryCard}
-      style={{ width: width }}
-    >
-      <CardHeader
-        className={classes.configurationCategoryCardHeader}
-        subheader={'Row ' + rowNum}
-      ></CardHeader>
+    <Card style={{ width: width }}>
```

Comment:
can we remove the width since we're going to be using css instead of inline styles?